src/Libraries/CoreNodes/CoreNodes.csproj

```diff
@@ -15,14 +15,22 @@
   <ItemGroup>
     <Compile Remove="GeometryColor.cs" />
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
     <Reference Include="Microsoft.CSharp" />
     <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.16.0.3357"/>
     <PackageReference Include="ncalc" Version="1.3.8">
       <!--Exclude copying the runtime dlls because we need to keep binary compatibility with Revit for now -->
       <ExcludeAssets>runtime</ExcludeAssets>
     </PackageReference>
     <Reference Include="PresentationCore" />
+  </ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+    <PackageReference Include="CoreCLR-NCalc" Version="2.2.92" />
```

Comment:
same as above, I don't think it's worth the complexity. Maybe we can even migrate to a codeblock or python.